packages/gestalt-codemods/0.108.0-0.109.0/__tests__/convert-font-weight.test.js

```diff
@@ -0,0 +1,17 @@
+import { defineTest } from 'jscodeshift/dist/testUtils.js';
+
+jest.mock('../convert-font-weight', () => {
+  return Object.assign(require.requireActual('../convert-font-weight'), {
+    parser: 'flow',
+  });
+});
+
+describe('convert-font-weight', () => {
+  [
+    'convert-font-weight-dynamic',
+    'convert-font-weight-spread',
+    'convert-font-weight-static',
```

Comment:
Do we also need to worry about any cases with `<Text bold={false} />`?

Reply:
Thanks - also added a test case for that (didn't find any in pinboard)